lib/Crypto/verifyrequest.h

```diff
@@ -33,7 +33,7 @@ class SAILFISH_CRYPTO_API VerifyRequest : public Sailfish::Crypto::Request
     Q_PROPERTY(Sailfish::Crypto::CryptoManager::SignaturePadding padding READ padding WRITE setPadding NOTIFY paddingChanged)
     Q_PROPERTY(Sailfish::Crypto::CryptoManager::DigestFunction digestFunction READ digestFunction WRITE setDigestFunction NOTIFY digestFunctionChanged)
     Q_PROPERTY(QString cryptoPluginName READ cryptoPluginName WRITE setCryptoPluginName NOTIFY cryptoPluginNameChanged)
-    Q_PROPERTY(bool verified READ verified NOTIFY verifiedChanged)
+    Q_PROPERTY(Sailfish::Crypto::CryptoManager::VerificationResult verified READ verified NOTIFY verifiedChanged)
```

Comment:
I'd suggest changing the name of this property to verification, and the getter/notif to verification()/verificationChanged()

Reply:
+1

Reply:
Sounds good

Reply:
or verificationStatus